docs/metrics/geometry-matching.md

```diff
@@ -172,10 +172,10 @@ the IoU threshold to be considered as a valid match.
 	8. Repeat 5-7 on the next inference;
 
 
-### COCO Detection Challenge
+### COCO
 
-[COCO detection challenge](https://cocodataset.org) evaluation has a couple more things to consider when matching
-inference geometries. First, similarly to how `difficult` ground truths are treated in Pascal VOC, COCO benchmark
+Microsoft's [COCO (Common Objects in Context)](https://cocodataset.org) evaluation has a couple more things to consider when matching
```

Comment:
More than just Microsoft — I'd remove this

Reply:
Ah the original dataset is from Microsoft. I see that the contributors now are more than Microsoft. Will remove